src/OSPSuite.Core/Domain/Builder/SimulationBuilder.cs

```diff
@@ -92,7 +92,8 @@ private void performMerge()
          _observers.AddRange(allBuilder(x => x.Observers));
          _molecules.AddRange(allBuilder(x => x.Molecules));
          _parameterValues.AddRange(allStartValueBuilder(x => x.SelectedParameterValues));
-         _initialConditions.AddRange(allStartValueBuilder(x => x.SelectedInitialConditions));
+         _initialConditions.AddRange(allStartValueBuilder(x => x.SelectedInitialConditions)
+            .Concat(_simulationConfiguration.ExpressionProfiles.SelectMany(expressionProfile => expressionProfile.InitialConditions)));
```

Comment:
consistency. Why not use x here where we have x everywhere. It feels like it;s something difference and it's not